app/renderer/actions/Inspector.js

```diff
@@ -432,23 +432,46 @@ export function searchForElement (strategy, selector) {
 }
 
 
-export function resetSearchForPORootElement () {
+export function resetSearchForPOElements () {
   return (dispatch) => {
     dispatch({type: SEARCHING_FOR_ROOT_ELEMENTS});
   };
 }
 
-export function searchForPORootElement (strategy, selector, po) {
+export function searchForPOElements (po, strategyMap) {
+  const [type, selector] = Object.entries(po.selector)[0];
+  let strategy = strategyMap[type];
+
   return async (dispatch, getState) => {
     try {
       const callAction = callClientMethod({strategy, selector, fetchArray: true});
```

Comment:
callClientMethod({strategyMap[type], selector, fetchArray: true});

Reply:
strategy is a property name of the object that take in as a parameter by callClientMethod. It cannot be replaced by just a value like strategyMap[type] here otherwise it won't pass the compiler.